src/pages/docs/data-folder.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Jan Data Folder
+description: Discover the Structure of Jan Data.
+sidebar_position: 2
+keywords:
+  [
+    Jan,
+    Customizable Intelligence, LLM,
+    local AI,
+    privacy focus,
+    free and open source,
+    private and offline,
+    conversational AI,
+    no-subscription fee,
+    large language models,
+    quickstart,
+    getting started,
+    using AI model,
+  ]
+---
+
+import { Tabs } from 'nextra/components'
+import { Callout, Steps } from 'nextra/components'
+
+# Jan Data Folder
+
+After installing Jan, a structured data folder is created on the local filesystem. This setup promotes an organized approach to data management. Below is a guide to understanding how files are organized within Jan data folder.
+
+## Folder Structure
+Jan app data folder should have the following folder structure:
+```yaml
+jan/                                
+  assistants/                           
+  engines/                              
```

Comment:
No more engines folder. Please: 
1. remove
2. replace with where the engine stuff went (in extensions?)